llvm/include/llvm/CodeGen/FunctionLoweringInfo.h

```diff
@@ -191,6 +191,7 @@ class FunctionLoweringInfo {
   /// Collection of dbg.declare instructions handled after argument
   /// lowering and before ISel proper.
   SmallPtrSet<const DbgDeclareInst *, 8> PreprocessedDbgDeclares;
+  SmallPtrSet<const DPValue *, 8> PreprocessedDPVs;
```

Comment:
What do you think about DPVDeclares, to distinguish this from "normal" DPValues? (Alas, naming things is one of the two difficult parts of CS).

Reply:
Do you have a preference between PreprocessedDPVDeclares and PreprocessedDPDeclares (no V, as in in DPV, in the second one)? The former, DPVDeclares, might be more grep-friendly?

Reply:
Let's go with the former, DPV indicating "debug-info things are happening now" is an alright (if not perfect) thing to maintain, "DP" by itself is open to more interpretation.